CONTRIBUTING.md

```diff
@@ -18,7 +18,20 @@ Additional resources specific to the packages repository:
   [**PRs need tests**](https://github.com/flutter/flutter/wiki/Tree-hygiene#tests), so
   this is critical to read before submitting a plugin PR.
 
-## Notes
+### Code review processes and automation
+
+Each package has a code owner (specified in 
+[CODEOWNERS](https://github.com/flutter/packages/blob/main/CODEOWNERS));
+PRs will automatically be assigned to the appropriate code owners for review.
+Review is expected within a week or two (but preferred much quicker than that!).
```

Comment:
This is redundant with https://github.com/flutter/flutter/wiki/Tree-hygiene (including the tl;dr at the top) which is part of the contributing guide we already point to. I would prefer to avoid redundant docs, especially given that I think the biggest problem our contributing docs have is that they are incredibly long.

Reply:
I mean, all I really need here is lines 28-31 but it will look mighty silly if that's the only text here.

Reply:
How about we collapse these first two paragraphs to just:

```
PRs will automatically be assigned to [code owners](https://github.com/flutter/packages/blob/main/CODEOWNERS) for review. If a code owner is creating a PR, they should explicitly pick a
[Flutter team member](https://github.com/flutter/flutter/wiki/Contributor-access)
as a code reviewer.
```

Reply:
updated, PTAL